csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html

```diff
@@ -31,6 +31,10 @@
       <a href="#prize">
         Prizes
```

Comment:
this page is not passing the [validator test](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-420%2Fevents%2Fmm%2F2021).

Reply:
@SpartanApple the validation is still having issues